addon/src/test-support/helpers/drag.js

```diff
@@ -108,4 +108,11 @@ export async function drag(mode, itemSelector, offsetFn, callbacks = {}) {
     await callbacks.dragend();
     await settled();
   }
+
+  await waitUntil(
```

Comment:
is this still needed? wouldn't settled state already be bring us to when there is no `.is-dropping`?

Reply:
the problem is, that there is no waiting for `transitionEnd`, for this reason we need to wait... only adding `await settled()` will not fix the problem 😒